src/pizzeria/Order.java

```diff
@@ -0,0 +1,84 @@
+package pizzeria;
+
+import java.time.LocalTime;
+import java.util.*;
+
+public class Order {
+    private final int orderNumber;
+    private final int customerNumber;
+    private final List<String> pizzaName;
+    private final String pizzaType;
+    private final List<Integer> quantityOfOrder;
+    private final LocalTime time = LocalTime.now();
+    private final Set<Ingredients> getFilledPizza = new HashSet<>();
+
+    Order(int customerNumber, List<String> pizzaName, List<Integer> quantityOfOrder, String pizzaType) {
+        Random order = new Random();
+        this.orderNumber = order.nextInt(100000);
+        this.customerNumber = customerNumber;
+        this.pizzaName = pizzaName;
+        this.quantityOfOrder = quantityOfOrder;
+        this.pizzaType = pizzaType;
+    }
+
+    public int getOrderNumber() {
+        return orderNumber;
+    }
+
+    public Set<Ingredients> getFillPizza() {
+        return getFilledPizza;
+    }
+
+    public int getCustomerNumber() {
+        return customerNumber;
+    }
+
+    public List<String> getPizzaName() {
+        int i = 0;
+        if (pizzaName.get(i).length() < 4 || pizzaName.get(i).length() > 20) {
+            pizzaName.set(i, "customer_name_" + getOrderNumber()+ " ");
+        }
+        return pizzaName;
+    }
+
+    public LocalTime getTime() {
+        return time;
+    }
+
+    public String getPizzaType() {
+        return pizzaType;
+    }
+
+    public List<Integer> getQuantityOfOrder() {
```

Comment:
This validation is necessary when you create an Order object. You don't need this validation when the order has already been created and you trying to get the quantity.